src/StackAdmin/Azs.Network.Admin/Tests/src/Common.ps1

```diff
@@ -12,12 +12,59 @@
 # limitations under the License.
 # ----------------------------------------------------------------------------------
 
-$global:TestName = ""
-
-if(-not $RunRaw) {
-		# Load the script block
-		$scriptBlock = { 
-			Get-MockClient -ClassName 'NetworkAdmin' -TestName $global:TestName
-		}
-		Mock New-ServiceClient $scriptBlock -ModuleName "Azs.Network.Admin"
+# Tests to skip
+$global:SkippedTests = $(
+    'TestGetAllVirtualNetworksOData',
```

Comment:
@deathly809 why are we skipping these tests?

Reply:
From what I remember we have an ODATA bug. Let me ask who worked on this.

Reply:
@cormacpayne Made a bug https://github.com/Azure/azure-powershell/issues/6664